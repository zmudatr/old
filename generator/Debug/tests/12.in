USTAW J uO
USTAW C pBEezwN
MIESZAJ J C
WYPISZ J
WYPISZ C
ZERUJ J
ZERUJ C
USTAW U 6CTuJcG
USTAW J aQdHtHAc_
MIESZAJ U J
WYPISZ U
WYPISZ J
ZERUJ U
ZERUJ J
USTAW Y IP
USTAW Q HaiR8DaLnD
MIESZAJ Y Q
WYPISZ Y
WYPISZ Q
ZERUJ Y
ZERUJ Q
USTAW L 5PvuT6tM
USTAW W BfEQ
MIESZAJ L W
WYPISZ L
WYPISZ W
ZERUJ L
ZERUJ W
USTAW V Tw
USTAW P kes3
MIESZAJ V P
WYPISZ V
WYPISZ P
ZERUJ V
ZERUJ P
USTAW O DdjgT
USTAW N q
MIESZAJ O N
WYPISZ O
WYPISZ N
ZERUJ O
ZERUJ N
USTAW Y 1_k
USTAW A H2
MIESZAJ Y A
WYPISZ Y
WYPISZ A
ZERUJ Y
ZERUJ A
USTAW Y 2L2Qjvn3lq
USTAW J uVoHTiV*
MIESZAJ Y J
WYPISZ Y
WYPISZ J
ZERUJ Y
ZERUJ J
USTAW T 0
USTAW H Tal9wjS9P
MIESZAJ T H
WYPISZ T
WYPISZ H
ZERUJ T
ZERUJ H
USTAW E IFrmLpMy
USTAW X WnDc1lgC7
MIESZAJ E X
WYPISZ E
WYPISZ X
ZERUJ E
ZERUJ X
USTAW T xaNZ1C8j
USTAW A +2I
MIESZAJ T A
WYPISZ T
WYPISZ A
ZERUJ T
ZERUJ A
USTAW X jkiSnbTotY
USTAW N eT4Y
MIESZAJ X N
WYPISZ X
WYPISZ N
ZERUJ X
ZERUJ N
USTAW U ZbxGwO
USTAW S IhrJp
MIESZAJ U S
WYPISZ U
WYPISZ S
ZERUJ U
ZERUJ S
USTAW S a8M*Qh83
USTAW C u
MIESZAJ S C
WYPISZ S
WYPISZ C
ZERUJ S
ZERUJ C
USTAW H l4G2um*E
USTAW Z /lnkyg0
MIESZAJ H Z
WYPISZ H
WYPISZ Z
ZERUJ H
ZERUJ Z
USTAW B AcoJq
USTAW M m9ZsHe
MIESZAJ B M
WYPISZ B
WYPISZ M
ZERUJ B
ZERUJ M
USTAW Z 0dqJK*Ko
USTAW J ol
MIESZAJ Z J
WYPISZ Z
WYPISZ J
ZERUJ Z
ZERUJ J
USTAW S oM*A7cD
USTAW F ig
MIESZAJ S F
WYPISZ S
WYPISZ F
ZERUJ S
ZERUJ F
USTAW I zHVwwvA
USTAW N y1u
MIESZAJ I N
WYPISZ I
WYPISZ N
ZERUJ I
ZERUJ N
USTAW H xffCV
USTAW O xZSoAMlNE
MIESZAJ H O
WYPISZ H
WYPISZ O
ZERUJ H
ZERUJ O
