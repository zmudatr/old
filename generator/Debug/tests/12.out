upOBEezwN

6aCQTduHJtcHGAc_

IHPaiR8DaLnD

5BPfvEuQT6tM

Tkwes3

DqdjgT

1H_2k

2uLV2oQHjTvinV3*lq

0Tal9wjS9P

IWFnrDmcL1plMgyC7

x+a2NIZ1C8j

jekTi4SYnbTotY

ZIbhxrGJwpO

au8M*Qh83

l/4lGn2kuymg*0E

Amc9oZJsqHe

0odlqJK*Ko

oiMg*A7cD

zyH1VuwwvA

xxfZfSCoVAMlNE

